diff --git a/crt/aws-crt-cpp/crt/aws-c-io/source/event_loop.c b/crt/aws-crt-cpp/crt/aws-c-io/source/event_loop.c
index 0ec298d..9b9fd8d 100644
--- a/crt/aws-crt-cpp/crt/aws-c-io/source/event_loop.c
+++ b/crt/aws-crt-cpp/crt/aws-c-io/source/event_loop.c
@@ -266,7 +266,7 @@ struct aws_event_loop_group *aws_event_loop_group_new_internal(
 
     el_group->allocator = allocator;
     aws_ref_count_init(
-        &el_group->ref_count, el_group, (aws_simple_completion_callback *)s_aws_event_loop_group_shutdown_async);
+        &el_group->ref_count, el_group, (aws_simple_completion_callback *)s_aws_event_loop_group_shutdown_sync);
 
     uint16_t el_count = options->loop_count;
     if (el_count == 0) {
