diff --git a/src/backends/odbc/statement.cpp b/src/backends/odbc/statement.cpp
--- a/src/backends/odbc/statement.cpp
+++ b/src/backends/odbc/statement.cpp
@@ -423,14 +423,12 @@
     case SQL_BIGINT:
         dbtype = is_unsigned == SQL_TRUE ? db_uint64 : db_int64;
         break;
-    case SQL_WCHAR:
-    case SQL_WVARCHAR:
-    case SQL_WLONGVARCHAR:
-        dbtype = db_wstring;
-        break;
     case SQL_CHAR:
     case SQL_VARCHAR:
     case SQL_LONGVARCHAR:
+    case SQL_WCHAR:
+    case SQL_WVARCHAR:
+    case SQL_WLONGVARCHAR:
     default:
         dbtype = db_string;
         break;
